mungegithub/mungers/approvers/approvers_test.go

```diff
@@ -309,13 +310,34 @@ func TestGetCCs(t *testing.T) {
 			// We don't suggest approvers for a and b, only for unapproved c.
 			expectedCCs: []string{"Carol"},
 		},
+		{
+			testName:  "A, B, C; Nothing approved, but assignees can approve",
+			filenames: []string{"a/test.go", "b/test.go", "c/test"},
+			testSeed:  0,
+			// Approvers are valid approvers, but not the one we would suggest
+			currentlyApproved: sets.NewString(),
+			assignees:         []string{"Art", "Ben"},
+			// We suggest assigned people rather than "suggested" people
+			expectedCCs: []string{"Art", "Ben", "Carol"},
+		},
+		{
+			testName:          "A, B, C; Nothing approved, but SOME assignees can approve",
+			filenames:         []string{"a/test.go", "b/test.go", "c/test"},
+			testSeed:          0,
+			currentlyApproved: sets.NewString(),
+			// Assignees are a mix of potential approvers and random people
+			assignees: []string{"Art", "Ben", "John", "Jack"},
+			// We suggest assigned people rather than "suggested" people
+			expectedCCs: []string{"Art", "Ben", "Carol"},
+		},
```

Comment:
one nit.  it'd be helpful to have a test that suggests Art and Bill when neither are assigned, but then when Ben is assigned it suggests Art and Ben